spec/models/blueprint_spec.rb

```diff
@@ -34,6 +34,10 @@
     FactoryGirl.create(:service_template, :name => 'Service Template Bundle', :display => true, :blueprint => subject)
   end
 
+  it 'is taggable' do
+    expect(subject).to respond_to(:tag_with)
```

Comment:
`acts_as_miq_taggable` is a mixin. Any behavior should be tested separately or may have been tested.